src/ObjectUtils/include/ObjectUtils/SymbolsFile.h

```diff
@@ -14,6 +14,11 @@
 
 namespace orbit_object_utils {
 
+struct ObjectFileInfo {
+  uint64_t load_bias = 0;
+  uint64_t executable_segment_offset = 0;
```

Comment:
I see the comments at other locations and am wondering if a comment here make sense too. Basically the same short explanation: this is called ImageBase in Coff. 

I like comments in headers, because this is usually where i look first when trying to figure out how to use classes.

Reply:
Good suggestion, will add short comments here.

Reply:
Done.